docker-cmd/start-docker-compose.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# make folders for docker-compose volumes for apache2
+mkdir -p dist/forus-platform.provider.general
+mkdir -p dist/forus-platform.sponsor.general
+mkdir -p dist/forus-platform.validator.general
+mkdir -p dist/forus-webshop-general.panel
+
+docker-compose up -d && docker-compose exec app sh -c "cd src && npm i && yes n | gulp init && gulp compile"
```

Comment:
during first run of this command got an error: 

```
aleksei@aleksei-pc:/var/www/forus-frontend$ ./docker-cmd/start-docker-compose.sh
Creating network "forus-frontend_forus-frontend" with driver "bridge"
Pulling app (node:14-alpine)...
14-alpine: Pulling from library/node
df9b9388f04a: Already exists
d5636aa35043: Pull complete
a35fb614082d: Pull complete
14e16d3b67f2: Pull complete
Digest: sha256:8845b4f88f64f8c56a39236648ba22946e806a6153c10911f77b70e5a2edb4ca
Status: Downloaded newer image for node:14-alpine
Pulling apache2 (httpd:2.4)...
2.4: Pulling from library/httpd
c229119241af: Already exists
1805d911aae4: Pull complete
e3709b515d9c: Pull complete
4f53b8f15873: Pull complete
3b60f356ab85: Pull complete
Digest: sha256:e3c40b99ffa305c6e52346a6618b1fb47ea0568c999b26f8900cd26febab1160
Status: Downloaded newer image for httpd:2.4
Creating forus-frontend-app     ... done
Creating forus-frontend-apache2 ... done
npm WARN jadeify@4.7.0 requires a peer of browserify@>= 2.4.0 < 15 but none is installed. You must install peer dependencies yourself.
npm WARN jadeify@4.7.0 requires a peer of jade@^1.9.2 but none is installed. You must install peer dependencies yourself.
npm WARN pdfjs-dist@2.2.228 requires a peer of webpack@^3.0.0 || ^4.0.0-alpha.0 || ^4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN worker-loader@2.0.0 requires a peer of webpack@^3.0.0 || ^4.0.0-alpha.0 || ^4.0.0 but none is installed. You must install peer dependencies yourself.
npm WARN quick-dev-template@0.2.1 No repository field.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.3.2 (node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.2.13 (node_modules/glob-watcher/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})

audited 1430 packages in 7.158s

70 packages are looking for funding
  run `npm fund` for details

found 14 vulnerabilities (1 low, 8 moderate, 4 high, 1 critical)
  run `npm audit fix` to fix them, or `npm audit` for details
sh: gulp: not found

```

second run finished successfully, did you have the same error??